### ISSUE TITLE
KSUID and ActiveRecord Associations

### ISSUE THREAD

**Issue by sidot3291** [michaelherold/ksuid-ruby#12]
```
class House
  include KSUID::ActiveRecord[:id]
  has_many :people
end

class Person
  include KSUID::ActiveRecord[:id]
  belongs_to :house
end

Person.new(house: House.last)
```

This throws an error by default.

**Comment by mattes** [comment]
Possibly related: https://stackoverflow.com/questions/60242514/rails-custom-activerecordtype-fails-when-using-class-name-in-has-many-throu

**Comment by michaelherold** [comment]
I believe this was solved by #30
